orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -66,6 +70,50 @@ func TestSpecServicer_NewSpecServicerFromFile(t *testing.T) {
 	assert.Equal(t, testStandaloneFileContents, standaloneRes.SwaggerSpec)
 }
 
+func TestSpecServicer_NewSpecServicerFromSpecsPartialErr(t *testing.T) {
+	service := "a service"
+
+	partialErrMock := &mocks.Specs{}
+	partialErrMock.On("GetPartialSpec", service).
+		Return("", errors.New("partial err"))
+	standaloneSpec := "standalone spec"
+	partialErrMock.On("GetStandaloneSpec", service).
+		Return(standaloneSpec, nil)
+
+	servicer := swagger.NewSpecServicerFromSpecs(partialErrMock, service)
+	assert.NotNil(t, servicer)
+
+	partialRes, err := servicer.GetPartialSpec(context.Background(), &protos.PartialSpecRequest{})
+	assert.NoError(t, err)
+	assert.Empty(t, partialRes.SwaggerSpec)
+
+	standaloneRes, err := servicer.GetStandaloneSpec(context.Background(), &protos.StandaloneSpecRequest{})
+	assert.NoError(t, err)
+	assert.Equal(t, standaloneSpec, standaloneRes.SwaggerSpec)
+}
+
+func TestSpecServicer_NewSpecServicerFromSpecsStandaloneErr(t *testing.T) {
+	service := "another service"
+
+	standaloneErrMock := &mocks.Specs{}
+	partialSpec := "partial spec"
+	standaloneErrMock.On("GetPartialSpec", service).
+		Return(partialSpec, errors.New("partial err"))
+	standaloneErrMock.On("GetStandaloneSpec", service).
+		Return("", nil)
```

Comment:
Here and above: I would go ahead and just not line break on these, they're short enough